### PR TITLE
ready for release of new version

### PR DESCRIPTION
**Pull request by BobBorges**

### CODE REVIEW

File: test/data/independent-mp.csv
Comment (ninpnin) [inline test/data/independent-mp.csv:1]: Why is this semicolon separated? (we might have discussed this before)

**Review by ninpnin — APPROVED** [review]
LGTM (once that one thing is addressed)

**Comment by BobBorges** [comment]
all (most?) of the test files are semi-colon separated. There were problems with commas inside the string at some point and pandas isn't good at that for some reason.

**Comment by ninpnin** [comment]
I see. I think I suggested using tabs (they are not characters we support in the text) but let's address that later.

Feel free to proceed.

**Pull request merged**